### PR TITLE
chore: update std version

### PR DESCRIPTION
**Pull request by kt3k**
This PR updates deno_std versions to pass the canary CI.

### CODE REVIEW

**Review by satyarohith — APPROVED** [review]

**Comment by kt3k** [comment]
hmm CI seems failing with deno 1.9.1, which seems incompatible with the latest deno_std...

Should we still support deno 1.9.1? :thinking:

**Comment by yeskunall** [comment]
On a complete side-note—seeing as it was recently done in #68 (although I now see I forgot to update `tests/deps.ts`)—have you guys considered a way to auto-update `deployctl`’s `deps.ts`?

Not saying that doing this manually for a few dependencies isn’t feasible, but it’d be nice to automate the process, IMO.

**Comment by kt3k** [comment]
@yeskunall 

> On a complete side-note—seeing as it was recently done in #68 (although I now see I forgot to update tests/deps.ts)—have you guys considered a way to auto-update deployctl’s deps.ts?

I don't have much experience with it yet, but hayd's udd seems promising to me for that purpose https://github.com/hayd/deno-udd

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Setup Deno
         run: |
-          curl -fsSL https://deno.land/x/install/install.sh | sh ${{ matrix.deno == 'old' && '-s v1.9.1' || '' }}
+          curl -fsSL https://deno.land/x/install/install.sh | sh ${{ matrix.deno == 'old' && '-s v1.11.0' || '' }}
```
Comment (kt3k) [inline .github/workflows/ci.yml:26]: It seems 1.11.0 is the oldest deno version which can pass the CI with the latest deno_std

**Comment by kt3k** [comment]
Let's try this

**Pull request merged**